cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id='viewing-pipeline-logs-using-the-openshift-logging-operator']
+= Viewing pipeline logs using the OpenShift Logging Operator
+
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+
+:context: viewing-pipeline-logs-using-the-openshift-logging-operator
+
+
+toc::[]
+
+
+The logs generated by pipeline runs, task runs, and event listeners are stored in the respective pods. It is often useful to review and analyze such logs for troubleshooting and audits. However, retaining the pods indefinitely leads to unnecessary resource consumption and cluttered namespaces.
+
+To eliminate dependency on pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator provided by {pipelines-title}. These Operators helps you see pipeline logs using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after deleting the pods containing the logs.
```

Comment:
```suggestion
To eliminate dependency on pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator provided by {pipelines-title}. You can see pipeline logs using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after deleting the pods that contain the logs.
```

Suggested edit. The second sentence was pretty hard to understand so I suggest the above edit. If it changes the meaning of the sentence, feel free to ignore.